docs/develop/support-ie-11.md

```diff
@@ -0,0 +1,50 @@
+---
+
+# Support Internet Explorer 11
+
+Office Add-ins are web applications that are displayed inside IFrames when running on Office on the web. Office Add-ins are displayed using embedded browser controls when running in Office on Windows or Office on the Mac. The embedded browser controls are supplied by the operating system or by a browser installed on the user's computer.
+
+If you plan to market your add-in through AppSource or you plan to support certain older versions of Windows and Office, your add-in must work in the embeddable browser control that is based on Internet Explorer 11 (called "Trident"). For information about which combinations of Windows and Office use the Internet Explorer 11 browser control, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
+
+> [!IMPORTANT]
+> Internet Explorer 11 doesn't support some HTML5 features such as media, recording, and location. If your add-in must support Internet Explorer 11 (see the second paragraph), then you can't use these features.
+
+Internet Explorer 11 doesn't support JavaScript versions later than ES5. If you want to use the syntax and features of ECMAScript 2015 or later, or TypeScript, you have two options as described in this article. (You can also combine these two techniques.)
+
+## Use a transpiler
+
+You can write your code in either TypeScript or in a version of JavaScript later than ES5 and then transpile it at build-time into ES5 JavaScript. The resulting ES5 files are what you upload to your add-in's web application.
+
+There are two popular transpilers. Both of them can work with source files that are TypeScript or post-ES5 JavaScript. They will both also work with React files (.jsx and .tsx).
+
+- [babel](https://babeljs.io/)
+- [tsc](https://www.typescriptlang.org/index.html)
+
+See the documentation for either of them for information about installing and configuring the transpiler in your add-in project. We recommend that you use a task runner, such as [Grunt](https://gruntjs.com/) or [WebPack](https://webpack.js.org/) to automate the transpilation. For examples of add-ins that use transpilers, open [Office Add-ins Patterns and Practices](https://github.com/OfficeDev/PnP-OfficeAddins) repo and search for “target: es5” to find samples that use tsc and search for "babel" to find samples that use babel.
+
+> [!NOTE]
+> If you are using Visual Studio (not Visual Studio Code), tsc is probably easiest to use. You can install support for it with a nuget package. For more information, see [JavaScript and TypeScript in Visual Studio 2019](/visualstudio/javascript/javascript-in-vs-2019). To use babel with Visual Studio, you could create a build script and/or use the Task Runner Explorer in Visual Studio and tools like the [WebPack Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.WebPackTaskRunner) or [NPM Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.NPMTaskRunner).
```

Comment:
The style guide asks to avoid "and/or". I also find the last part of the last sentence to be confusing because of the "and", so I'm wondering if "with" makes a clearer association of the TaskRunner tools with the explorer.
```suggestion
> If you are using Visual Studio (not Visual Studio Code), tsc is probably easiest to use. You can install support for it with a nuget package. For more information, see [JavaScript and TypeScript in Visual Studio 2019](/visualstudio/javascript/javascript-in-vs-2019). To use babel with Visual Studio, create a build script or use the Task Runner Explorer in Visual Studio with tools like the [WebPack Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.WebPackTaskRunner) or [NPM Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.NPMTaskRunner).
```